plugins/ImageGraph/API.php

```diff
@@ -238,8 +238,11 @@ public function get(
 
             $ordinateColumns = array();
             if (empty($columns)) {
-                $ordinateColumns[] =
-                    empty($reportColumns[self::DEFAULT_ORDINATE_METRIC]) ? key($metadata['metrics']) : self::DEFAULT_ORDINATE_METRIC;
+                if (!empty($metadata['metrics']) && empty($reportColumns[self::DEFAULT_ORDINATE_METRIC])) {
+                    $ordinateColumns[] = key($metadata['metrics']);
+                } else {
+                    $ordinateColumns[] = self::DEFAULT_ORDINATE_METRIC;
+                }
```

Comment:
This might actually cause issues when rendering the graph if the metric `self::DEFAULT_ORDINATE_METRIC` isn't available. We should consider throwing an exception if `$reportColumns[self::DEFAULT_ORDINATE_METRIC]` is empty or at least logging an error